getting_started/step_by_step/your_first_game.rst

```diff
@@ -865,49 +843,60 @@ Note that a new instance must be added to the scene using ``add_child()``.
 
     func _on_MobTimer_timeout():
         # Choose a random location on Path2D.
-        $MobPath/MobSpawnLocation.offset = randi()
+        var mob_spawn_location = get_node("MobPath/MobSpawnLocation");
+        mob_spawn_location.offset = randi()
```

Comment:
What is the point of this?

Reply:
This same node is referenced 2 other times in the script below these lines, so this means that we don't need to get it thrice.